src/_data/schemas/lpa.json

```diff
@@ -0,0 +1,289 @@
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Person To Notify"
+        },
+        "ReplacementAttorney": {
+            "type": "object",
+            "description": "The Replacement Attorney.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Replacement Attorney"
+        },
+        "Actor": {
+            "type": "object",
+            "description": "Generic Actor that can be inherited across definitions.",
+            "additionalProperties": false,
+            "properties": {
+                "id": {
+                    "title": "Actor unique identifier",
+                    "description": "Identifies the Actor with a unique identifier.",
+                    "type": "integer",
+                    "format": "^[0-9]+$"
+                },
+                "title": {
+                    "title": "Title",
+                    "description": "The title given to an actor.",
+                    "type": "string"
+                },
+                "firstNames": {
+                    "title": "First Name",
+                    "description": "The actors first name.",
+                    "type": "string"
+                },
+                "lastName": {
+                    "title": "Last Name",
+                    "description": "The actors last name.",
+                    "type": "string"
+                },
+                "otherNames": {
+                    "title": "Other Names",
+                    "description": "Any other names for the actor.",
+                    "type": "string"
+                },
+                "dateOfBirth": {
+                    "type": "integer",
+                    "title": "Date of Birth",
+                    "maxLength": 13,
+                    "pattern": "^[0-9]{13}$",
+                    "description": "A 13 digit timestamp. Must be over the age of 18."
+                },
+                "address": {
+                    "title": "Address",
+                    "description": "The full address of the Actor.",
+                    "$ref": "#/definitions/Address"
+                },
+                "emailAddress": {
+                    "title": "Email Address",
+                    "description": "The email address of the actor.",
+                    "type": "string"
+                }
+            },
+            "required": [
+                "firstName",
```

Comment:
Rather than "firstName" and "lastName" I'd prefer we went for terms like "givenName" and "familyName", as these are not Western-centric.

Reply:
> Are you alright if I extract all those points into a separate doc/ticket as all that stuff is gold and we can begin to add it all in as we build it up in other tickets?

Sure, as long as we capture the issues somehow. 

I think something to consider as another ticket is a top level wrapper (envelope-pattern like) that identifies which version of the LPA schema this LPA was created with, as the nature of the length of storage and changes over time will mean tracking that becomes important.